index.html

```diff
@@ -201,8 +201,27 @@ <h3>Skills</h3>
           </ul>
         </section>
       </section>
+
+      <footer class="contact-wrapper">
+        <h2 class="contact-title">Contact me</h2>
+        <p class="contact-paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="#" method="POST">
+
+          <ul class="form-content">
+            <li> <label><input type="text" name="fullname" placeholder="Full name"
+              maxlength="30" required></label></li>
+            <li><label><input type="email" name="email" placeholder="Enter your E-Mail address"></label></li>
+            <li><label><textarea name="message" id="msg"
+              placeholder="Write your message here" maxlength="500" required></textarea></label></li>
+
+            <li><button class="button-contact" type="submit">Get in Touch</button></li>
+          </ul>
```

Comment:
- [ ] Please add some margin to the elements to better match the template, they are currently sticked to each other.